src/main/java/net/minecraftforge/fmp/capabilities/ISlottedCapabilityProvider.java

```diff
@@ -0,0 +1,14 @@
+package net.minecraftforge.fmp.capabilities;
+
+import net.minecraft.util.EnumFacing;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.fmp.multipart.PartSlot;
+
+public interface ISlottedCapabilityProvider
+{
+    
+    public boolean hasCapability(Capability<?> capability, PartSlot slot, EnumFacing facing);
+
+    public <T> T getCapability(Capability<T> capability, PartSlot slot, EnumFacing facing);
```

Comment:
Please keep ISlottedCapabilityProvider. It's very useful if you're a wire on a side of a block and only want to read things which are on the same side on another block.